lib/src/git.rs

```diff
@@ -157,8 +183,12 @@ pub fn import_refs(
     }
 
     // Find commits that are no longer referenced in the git repo and abandon them
-    // in jj as well.
-    let new_git_heads = new_git_heads.into_iter().collect_vec();
+    // in jj as well. We must remove non-existing commits from new_git_heads, as
+    // they could have came from branches which were never fetched.
```

Comment:
nit: s/came/come/ (but maybe this change is made obsolete by my comment above)

Reply:
Done.